city_scrapers/spiders/atconj_County_Commission.py

```diff
@@ -0,0 +1,106 @@
+import json
+
+import scrapy
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.parser import parse as dateparse
+from scrapy import Selector
+
+
+class AtconjCountyCommissionSpider(CityScrapersSpider):
+    name = "atconj_County_Commission"
+    agency = "Atlantic County Board of County Commissioners"
+    timezone = "America/New_York"
+
+    original_url = "https://www.atlanticcountynj.gov"
+    meetings_url = "https://www.atlanticcountynj.gov/government/county-government/board-of-county-commissioners/meeting-schedule-agendas-and-minutes/-toggle-all"  # noqa
+
+    custom_settings = {
+        "ROBOTSTXT_OBEY": False,
+    }
+
+    default_location = {
+        "name": "Stillwater Building",
+        "address": "Stillwater Building, 201 S. Shore Road Northfield, New Jersey 08225",  # noqa
+    }
```

Comment:
Please exclude the "name" text from "address" text. Same exclusion in `_parse_location()` function
```python
default_location = {
    "name": "Stillwater Building",
    "address": "201 S. Shore Road Northfield, New Jersey 08225",
}
```